tests/contrib/anthropic/cassettes/anthropic_completion_error_async.yaml

```diff
@@ -0,0 +1,67 @@
+interactions:
```

Comment:
Nit - can we use the same cassettes for both sync and async tests to minimize cassette files? Thanks

Reply:
done